Source/Diagnostics/BTDiagnostics.H

```diff
@@ -37,6 +38,8 @@ private:
     bool m_plot_raw_fields_guards = false;
     /** Read relevant parameters for BTD */
     void ReadParameters ();
+    /** Determines timesteps at which full diagnostics are written to file */
```

Comment:
Is it "BTD" instead of "full" here?
```suggestion
    /** Determines timesteps at which BTD diagnostics are written to file */
```